source/Playnite/ViewModels/MainViewModelBase.cs

```diff
@@ -586,28 +586,28 @@ public async Task UpdateLibrary(LibraryPlugin plugin)
                 var addedGames = ImportLibraryGames(plugin, token);
                 if (AppSettings.ScanLibInstallSizeOnLibUpdate)
                 {
-                    UpdateLibraryInstallSizes(token);
+                    UpdateGamesInstallSizes(token, Database.Games.ToList(), LOC.ProgressScanningGamesInstallSize);
                 }
 
                 return addedGames;
             }, AppSettings.DownloadMetadataOnImport);
         }
 
-        public void UpdateLibraryInstallSizes(CancellationToken token)
+        public void UpdateGamesInstallSizes(CancellationToken token, List<Game> games, string progressMessageLocKey)
         {
             try
             {
                 ProgressActive = true;
                 ProgressValue = 0;
-                ProgressTotal = Database.Games.Count + 1;
+                ProgressTotal = games.Count + 1;
 
                 Logger.Info($"Starting Library Install Size scan");
-                ProgressStatus = Resources.GetString(LOC.ProgressScanningGamesInstallSize);
+                ProgressStatus = Resources.GetString(progressMessageLocKey);

```

Comment:
I'm not sure this is needed to have it configurable, if general string is "Scanning size of imported games…" it should be universal enough for all scenarios.

Reply:
The issue is that "Imported" would only apply to the actions that this PR added support because it scans the size of games that were newly imported only. In the case of library updates, the same string with "imported" would not apply because it not only scans the size of newly imported games but also of games that are already in library.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3100/commits/bfff4cae6d5ef7773c8616f178a170d5022f690f